crates/ty_python_semantic/resources/mdtest/narrow/type.md

```diff
@@ -127,7 +127,7 @@ class B: ...
 
 def _[T](x: A | B):
     if type(x) is A[str]:
```

Comment:
Unrelated but this is wrong, `type(x)` would only ever return `A`.

Reply:
Yes, in this case we should ignore the narrowing (and potentially also treat it as a known-false condition and unreachable code); the test will never pass.